bundles/mapping/mapmodule/plugin/getinfo/GetInfoPlugin.js

```diff
@@ -277,12 +277,15 @@ Oskari.clazz.define(
          *            lonlat coordinates
          */
         handleGetInfo: function (lonlat, layers) {
-            var me = this,
+            const me = this,
                 dteMs = (new Date()).getTime(),
                 layerIds = me._buildLayerIdList(layers || this.getSandbox().findAllSelectedMapLayers()),
                 mapVO = me.getSandbox().getMap(),
                 px = me.getMapModule().getPixelFromCoordinate(lonlat);
 
+            const timeseriesLayer = layers.find(layer => !!layer.getParams().time);
+            const time = timeseriesLayer.getParams().time;
```

Comment:
Gather additional params per layer to an object like `{ layerId : { time: timeseriesvalue } }`
```suggestion
        const additionalParams = layers
            .filter(layer => layerIds.include(layer.getId()))
            .reduce((result, layer) => {
                const params = layer.getParams();
                if (typeof params !== 'object' || !Object.keys(params).length) {
                    return result;
                }
                result[layer.getId()] = params;
                return result;
            }, {});
```